packages/atomic/src/components/atomic-result-placeholder/atomic-result-placeholder.tsx

```diff
@@ -21,9 +21,14 @@ export class AtomicResultPlaceholder {
   @Prop() display!: ResultDisplayLayout;
   @Prop() density!: ResultDisplayDensity;
   @Prop() imageSize?: ResultDisplayImageSize;
+  @Prop() classes = '';
```

Comment:
Instead of having both display/density/imageSize and `classes`, I would prefer to only have one of the two patterns. `classes` could be replaced with `isChild?: boolean` or `display/density/imageSize` could be removed and `classes` could be used instead